src/state/actions/dispatchArIOInteraction.ts

```diff
@@ -49,19 +47,12 @@ export default async function dispatchArIOInteraction({
   dispatch: Dispatch<TransactionAction>;
   signer?: ContractSigner;
   ao?: AoClient;
-  antAo?: AoClient;
   hyperbeamUrl?: string;
   scheduler?: string;
   fundFrom?: FundFrom | 'fiat';
   turboArNSClient?: TurboArNSClient;
 }): Promise<ContractInteraction> {
   let result: AoMessageResult<MessageResult | unknown> | undefined = undefined;
-  const aoCongestedTimeout = setTimeout(
-    () => {
-      eventEmitter.emit('network:ao:congested', true);
-    }, // if it is taking longer than 10 seconds, consider the network congested
-    1000 * 10,
-  );
```

Comment:
this is not the right place for this

Reply:
I see you removed it but didn't add it back in the workflows. Agree this should be configured better, ideally surrounding signing.